### DISCUSSION TITLE
Which signature algorithms to enable by default

### DISCUSSION THREAD

**Discussion by baentsch** [open-quantum-safe/oqs-provider#465] [General]
As [is known](https://github.com/open-quantum-safe/oqs-provider/issues/399) more than 64 active signature algorithms can break interaction with TLS servers. With the activation of Mayo, oqsprovider by default now has 58 algorithms active. The addition of Cross will mean we have to be selective which algorithms to have active by default to not wreak havoc on `openssl` servers.

This issue is to have a discussion which algorithms (and variants) thereof should be active by default with priority.

**Comment by SWilson4** [comment]
For a start, I think I would be inclined to make only one of "Falcon" and "Falcon-padded" available by default. Probably the padded version. It looks to me like that would trim off five algorithms. Perhaps when the final standard for ML-DSA comes out we can also switch off Dilithium (the round 3 version) by default, which would buy us another seven.

**Reply by baentsch** [reply]
Thanks for the good proposal, @SWilson4 . As it's the only one and I agree with it, changing this to an issue for implementation.
